mysql-test/suite/sys_vars/r/innodb_default_encryption_key_id_basic.result

```diff
@@ -1,3 +1,35 @@
 SELECT @@global.innodb_default_encryption_key_id;
 @@global.innodb_default_encryption_key_id
 0
+# restart:--early-plugin-load=keyring_file=keyring_file.so --keyring_file_data=MYSQL_TMP_DIR/keyring/keyring --default-table-encryption=ONLINE_TO_KEYRING --innodb-encryption-rotate-key-age=15 --innodb-encryption-threads=4 --innodb-default-encryption-key-id=4294967295 --log-error=MYSQL_TMP_DIR/my_restart.err
+# restart:--early-plugin-load=keyring_file=keyring_file.so --keyring_file_data=MYSQL_TMP_DIR/keyring/keyring --default-table-encryption=ONLINE_TO_KEYRING --innodb-encryption-rotate-key-age=15 --innodb-encryption-threads=4 --innodb-default-encryption-key-id=-1 --log-error=MYSQL_TMP_DIR/my_restart.err
+SET innodb_default_encryption_key_id=4294967295;
+Warnings:
+Warning	1292	Truncated incorrect innodb_encryption_key_id value: '4294967295'
```

Comment:
after every SET, please show the current value and global SELECT @@session. and @@global.